cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -197,20 +197,21 @@ public KafkaReconciler(
         this.operationTimeoutMs = config.getOperationTimeoutMs();
         this.kafkaNodePoolCrs = nodePools;
 
+        boolean isKRaftEnabled = config.featureGates().useKRaftEnabled() && ReconcilerUtils.kraftEnabled(kafkaCr);
```

Comment:
This expression same expression is being used in the file above. Perhaps we can make encapsulate it in method of ReconcilerUtils, i.e. add `ReconcilerUtils.kraftEnabled(featureGates, kafkaCr)`?

Reply:
Yeah, I think it makes totally sense.

Reply:
Actually this is the same we have for node pools. Not having a dedicated method helps to see immediately you are checking both feature gate being enabled and annotation being set, which could be hidden behind just one method. It's not a big deal, I would leave as it is.